.nuspec

```diff
@@ -1,29 +0,0 @@
-<?xml version="1.0"?>
```

Comment:
Why did you delete this file? Can't publish new nuget package without it.

Reply:
The Visual Studio 2019 don't really need the file for package to nuget format. It is included in the csproj file.

Reply:
Since I wrote this lib I moved to mac and don't have visual studio installed so please keep this file because I'm using myget service to publish new versions.